src/gdcdictionary/schemas/exposure.yaml

```diff
@@ -247,6 +252,7 @@ properties:
       - "Yes"
       - "No"
       - Unknown
+      - Not Reported
```

Comment:
Just realized this was deprecated, but not a big deal if this is kept in, we just need to remove it from the release notes.